backend/src/main/scala/cromwell/backend/validation/RuntimeAttributesValidation.scala

```diff
@@ -84,7 +84,7 @@ object RuntimeAttributesValidation {
     new RuntimeAttributesValidation[ValidatedType] {
       override def key: String = validation.key
 
-      override def coercion: Traversable[WomType] = validation.coercion
+      override def coercion: Iterable[WomType] = validation.coercion
```

Comment:
I probably would've used `List`..but you might have a specific reason using `Iterable`

Reply:
I was just trying to stay as close to the original as possible with `Traversable` being gone 🙂